bin/tools/utils.js

```diff
@@ -203,17 +195,17 @@ require(path.join(__dirname, "compiled", "node", "build", "compiler"));
    *  it does not matter if they use source or build, just make sure it is up to date
    */
   console.log("Compiling source version");
-  result = await runCommand(".", "./tmp/qx", "compile");
+  result = await runCommand(".", "node", "./tmp/qx", "compile", "--clean", "-M");
```

Comment:
@johnspackman : Would'nt it be enough to build the compiler?

Reply:
I think we only need the compiler at this stage - the testing happens using the re-compiled compiler, rather than the tmp compiler.  Please can you remove the `-M`?